src/pal/src/numa/numa.cpp

```diff
@@ -208,11 +206,16 @@ NUMASupportInitialize()
 #endif // HAVE_NUMA_H
     {
         // No NUMA
-        g_cpuCount = GetNumberOfProcessors();
+        g_cpuCount = PAL_GetLogicalCpuCountFromOS();
         g_groupCount = 1;
         g_highestNumaNode = 0;
 
         AllocateLookupArrays();
+
+        for (int i = 0; i < g_cpuCount; i++)
+        {
+            g_cpuToAffinity[i].Number = i;
+        }
```

Comment:
Yes, you are right. Thank you for spotting this. I was rushing the original change out before the feature complete date and this is a lesson for me to not to rush things. Could you please add setting the Group member to zero?